ocaml/src/maintenance_config.ml

```diff
@@ -113,5 +163,15 @@ module Update = struct
                auto_repair_disabled_nodes);
         enable_rebalance = Option.get_some_default
                              enable_rebalance
-                             enable_rebalance'; }
+                             enable_rebalance';
+        cache_eviction_prefix_preset_pairs =
+          List.filter
+            (fun pair ->
+             not (List.mem
+                    pair
+                    remove_cache_eviction_prefix_preset_pairs))
+            (List.rev_append
+               add_cache_eviction_prefix_preset_pairs
```

Comment:
Then objects in it will be evicted at twice the speed of others ... I'll make a change prevent this from happening